cocotb/_build_libs.py

```diff
@@ -248,7 +251,7 @@ def _get_common_lib_ext(include_dir, share_lib_dir):
             os.path.join(share_lib_dir, "gpi", "GpiCbHdl.cpp"),
             os.path.join(share_lib_dir, "gpi", "GpiCommon.cpp"),
         ],
-        extra_link_args=_extra_link_args("libgpi"),
+        extra_link_args=_extra_link_args(lib_name="libgpi", rpath="$ORIGIN"),
```

Comment:
Any idea why this needs `lib_name` but the ones below do not?

Reply:
On osx it seems to work the way this needs to be set on dependencies only. Not really hudge expert here.

Reply:
Are we sure is still needs to be set now that we set the rpath on these libraries?

Reply:
I can set. It should not harm to set it.

Reply:
Should I do ?

Reply:
I'm suggesting the reverse. What happens if you _don't_ set it?

Reply:
It looks it does not work https://travis-ci.org/github/themperek/cocotb/jobs/672471998 on osx.

Reply:
Thanks for trying